### ISSUE TITLE
seek support for http://lunacommons.org

### ISSUE THREAD

**Issue by dancorpse** [lovasoa/dezoomify#133]
http://lunacommons.org/luna/servlet/detail/AMICO~1~1~59317~91911:Woman-with-Children-in-an-Interior# , what is the zoom tool for this site?

**Comment by lovasoa** [comment]
They have a custom zoom tool.Look at the URL of their tiles:
`http://amico.davidrumsey.com/mrsid/bin/image_jpeg.pl?client=amico&image=Sids/FASF/fasf.64766.sid&x=1875&y=4015&width=750&height=530&level=0`

I think you can have the server return you the full image just by playing with the parameters in the URL


![large image](http://amico.davidrumsey.com/mrsid/bin/image_jpeg.pl?client=amico&image=Sids/FASF/fasf.64766.sid&x=1000&y=975&width=3500&height=5500&level=0)

**Comment by dancorpse** [comment]
Yes, I've tried out the full size image by putting random figures in the url after x= and y= and width= and height=, however I don't know what's the relationship between them.
